### PR TITLE
Restructure directory.

### PR DESCRIPTION
**Pull request by irvifa**
Create a Dockerfile to deploy the apps, and restructure the directory.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-baseline = ENV.get_template("./template_message.j2")
+# Load Jinja template.
+baseline = ENV.get_template()
 
 # flask
 app = Flask(__name__) 
 
 # telegram bot
 bot = telegram.Bot('TOKEN')
+chat_id = os.getenv('CHATID')
```
Comment (irvifa) [inline src/main.py:19]: @zufardhiyaulhaq probably we need to provide the env var in the K8s Job.

Comment (zufardhiyaulhaq) [reply src/main.py:19]: should we provide the env var for telegram bot token to?

Comment (irvifa) [reply src/main.py:19]: yes.. I think better that way, we can get the env var from the secret we provided

**Comment by irvifa** [comment]
possible improvement https://github.com/cloudnative-id/kubeweekly-telegram-bot/issues/2

File: src/main.py (hunk 1 above)
Comment (irvifa) [reply src/main.py:19]: this is addressed as well..

**Comment by irvifa** [comment]
@zufardhiyaulhaq by the way `data = request.get_json()` what does this line is doing? Wondering if we can manage to integrate this as the input json from the scrapper or read a file cc @2pai

**Comment by zufardhiyaulhaq** [comment]
> @zufardhiyaulhaq by the way `data = request.get_json()` what does this line is doing? Wondering if we can manage to integrate this as the input json from the scrapper or read a file cc @2pai

@irvifa Parses the incoming JSON request data

**Pull request merged**